src/Adapter/Manufacturer/CommandHandler/DeleteManufacturerLogoImageHandler.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Adapter\Manufacturer\CommandHandler;
+
+use ImageType;
+use PrestaShop\PrestaShop\Core\Domain\Manufacturer\Command\DeleteManufacturerLogoImageCommand;
+use PrestaShop\PrestaShop\Core\Domain\Manufacturer\CommandHandler\DeleteManufacturerLogoImageHandlerInterface;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Handles command which deletes manufacturer cover image using legacy object model
+ */
+class DeleteManufacturerLogoImageHandler extends AbstractManufacturerCommandHandler implements DeleteManufacturerLogoImageHandlerInterface
+{
+    /**
+     * @var string
+     */
+    protected $imageDir;
+
+    /**
+     * @var string
+     */
+    protected $tmpImageDir;
+
+    public function __construct(string $imageDir, string $tmpImageDir)
+    {
+        $this->imageDir = $imageDir;
+        $this->tmpImageDir = $tmpImageDir;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function handle(DeleteManufacturerLogoImageCommand $command): void
+    {
+        $fs = new Filesystem();
+
+        $imageTypes = ImageType::getImagesTypes('manufacturers');
```

Comment:
Perhaps migrate this method s we don't use any Legayc code in this feature ? :)

Reply:
As `ImageType::getImagesTypes` in a lot of place in `src` directory, it should be migrated in a specific PR. 

Ping @matthieu-rolland with the `New Image management system` Working Group.